### PR TITLE
Rename some Protobuf structures in the starrocks.lake package

### PR DESCRIPTION
**Pull request by sduzh**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [x] refactor
- [ ] others

### CODE REVIEW

File: gensrc/proto/starlake.proto [hunk 1]
```diff
@@ -7,7 +7,7 @@ option java_package = "com.starrocks.lake.proto";
 
 import "tablet_schema.proto";
 
-message RowsetMetaPB {
+message RowsetMetaDataPB {
```
Comment (imay) [inline gensrc/proto/starlake.proto:10]:
```suggestion
message RowsetMetadataPB {
```
https://en.wikipedia.org/wiki/Metadata

Comment (wyb) [inline gensrc/proto/starlake.proto:10]: Why add `Data`?

Comment (sduzh) [reply gensrc/proto/starlake.proto]: The word "meta" is an adjective.

Comment (sduzh) [reply gensrc/proto/starlake.proto]: updated

Comment (sduzh) [reply gensrc/proto/starlake.proto]: https://en.wikipedia.org/wiki/Metadata

**Review by imay — APPROVED** [review]

**Comment by sduzh** [comment]
run starrocks_fe_unittest

**Review by wyb — APPROVED** [review]

**Comment by sduzh** [comment]
run starrocks_fe_unittest

**Comment by sduzh** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by sduzh** [comment]
run starrocks_be_unittest

**Comment by sduzh** [comment]
run starrocks_fe_unittest

**Pull request merged**